app/src/main/java/com/example/drawingApp/ColorPickerUtil.kt

```diff
@@ -0,0 +1,201 @@
+package com.example.drawingApp
+
+import android.annotation.SuppressLint
+import android.content.Context
+import android.graphics.Color
+import android.view.LayoutInflater
+import android.widget.TextView
+import androidx.core.graphics.toColor
+import androidx.core.widget.addTextChangedListener
+import com.google.android.material.bottomsheet.BottomSheetDialog
+import kotlin.math.max
+import kotlin.math.min
+
+object ColorPickerUtil {
+
+    data class ColorPack(
+        val hue: Float,
+        val saturation: Float,
+        val value: Float,
+        val previousColor: Color,
```

Comment:
This is silly you have an Hsv object lol

Reply:
implemented